### PR TITLE
test to check if there is a submit button on the form.

### PR DESCRIPTION
**Pull request by fadhili-ed**
test checks for the presence of submit in the form whose class name was 'deposit_form'

### CODE REVIEW

**Comment by sigu** [comment]
@fadhili-ed how comes only one change is visible here?

File: cypress/integration/home_page_spec.js [hunk 1]
```diff
@@ -2,5 +2,9 @@ describe('Deposits Page', () => {
     it('successfully loads', () => {
         cy.visit('http://localhost:9292/deposits')
         cy.contains('Deposits')
+<<<<<<< HEAD
```
Comment (sigu) [inline cypress/integration/home_page_spec.js]: These are conflict markers, they should be removed when resolving conflicts

**Review by sigu — CHANGES_REQUESTED** [review]

File: cypress/integration/home_page_spec.js (hunk 1 above)
Comment (fadhili-ed) [reply cypress/integration/home_page_spec.js]: okay, so does that mean i remove the indicated lines?

File: app.rb [hunk 2]
```diff
@@ -3,7 +3,10 @@
 get '/' do 
     slim :home
 end
+<<<<<<< HEAD
```
Comment (sigu) [inline app.rb]: REmove this markers

File: app.rb [hunk 3]
```diff
@@ -3,7 +3,10 @@
...
 
 get '/deposits' do
     slim :deposits
-end
\ No newline at end of file
+end
+=======
```
Comment (sigu) [inline app.rb]: These markers too

**Review by sigu — CHANGES_REQUESTED** [review]
You can also remove the `cypress/integreation/examples` folder since we dont need the example tests on our project

File: app.rb (hunk 2 above)
Comment (fadhili-ed) [reply app.rb]: sawa, let me do that

File: cypress/integration/home_page_spec.js (hunk 1 above)
Comment (sigu) [reply cypress/integration/home_page_spec.js]: Yes, that is how to resolve some conflicts. Next time you get a conflict ask for video help if you not sure what to do 😄

Comment (fadhili-ed) [reply cypress/integration/home_page_spec.js]: sawa

**Comment by fadhili-ed** [comment]
this is my current cypress test code:
Screenshot from 2020-04-15 11-37-50

**Comment by fadhili-ed** [comment]
and this is my current page:
Screenshot from 2020-04-15 11-42-25

File: views/layout.slim [hunk 4]
```diff
@@ -3,6 +3,7 @@ html
     head
         title My Expenditure App
         link rel="stylesheet" type="text/css" href="css/style.css"
+        script src = "javascript/main.js"
```
Comment (sigu) [inline views/layout.slim]: It is usually advisable to have your js file load last so it does not block the rest of the page from rendering.

File: views/deposits.slim [hunk 5]
```diff
@@ -1,4 +1,9 @@
...
-    input type="text" placeholder="Amount Deposited"
-    button type="button" submit
\ No newline at end of file
+div id = "form_contents"
+    form class = "deposits_form"
+        input type="text" placeholder="Name" 
+        br    
+        input type="text" placeholder="Amount Deposited"
+        br
+        button [onclick="window.location.href = '/deposits' " type="button" ]
```
Comment (sigu) [inline views/deposits.slim]: what were we supposed to do on click 🤔 ? This was our step 0 if you remember well

aww-board (1)

File: views/layout.slim (hunk 4 above)
Comment (fadhili-ed) [reply views/layout.slim]: should i put it on the last line?

File: views/deposits.slim (hunk 5 above)
Comment (fadhili-ed) [reply views/deposits.slim]: yes, but should it not be on the button?

Comment (fadhili-ed) [reply views/deposits.slim]: my problem was how to call the function from here where the button exists

**Pull request merged**